javascripts/controllers/NewCtrl.js

```diff
@@ -1,5 +1,16 @@
 "use strict";
 
-app.controller("NewCtrl", function ($scope) {
-    $scope.controller = "NewCtrl";
+app.controller("NewCtrl", function ($location, $rootScope, $scope, ContactService) {
+
+    $scope.submitForm = () => {
```

Comment:
should this line have an input?

Reply:
I'm guessing....yes? I am unsure as to what it should be.

Reply:
Should it be contact?